study.md

```diff
@@ -57,7 +57,7 @@ Which ActiveRecord method creates new objects? Does this method persist objects
 as rows in the database, or is there another method required for persistence?
 
 ```md
-<!-- your response here -->
+.create. I'm unclear bout the second part of this question, but I believe that since it is a class method, one would not be able to persist the object.
```

Comment:
`#create` will instantiate a new object and save it to the database.

`#new` instantiates a new object but doesn't save it to the database.  `#save` needs to be invoked on it afterward to save it.

You didn't need to answer the second part since you answered with `#create`.